### PR TITLE
Display

### PR DESCRIPTION
**Pull request by 00Halfman00**
A first person visits the site, enters  a username in the home/lobby page, and is sent to waiting room; 
A second person does the aforementioned, too, but on second person entering a message, they are both sent to drawing game.

### CODE REVIEW

File: client/components/Lobby.jsx [hunk 1]
```diff
@@ -14,32 +15,38 @@ class Lobby extends Component {
     ev.preventDefault();
     const { player } = this.state;
     socket.name = player.name;
-    this.props.history.push('/waitingroom');
+    const { name } = socket;
+    socket.emit('change name', name);
```
Comment (mavirex) [inline client/components/Lobby.jsx]: minor suggestion, but I think we can condense these two lines into one, by deleting line 18 and changing line 19 to 
socket.emit('change name', player.name)

File: client/components/Lobby.jsx [hunk 2]
```diff
@@ -14,32 +15,38 @@ class Lobby extends Component {
...
   render() {
     const { name } = this.state;
-
-    return (
+    const { socket } = this.props;
+    return (socket.name === '' ? (
       <div>
         <h1>Lobby Room</h1>
         <div className="lobby-div">
+          {/* <form className="lobby-form" onSubmit={this.save}> */}
```
Comment (mavirex) [inline client/components/Lobby.jsx]: I think we can delete this commented out line

File: client/components/Lobby.jsx [hunk 3]
```diff
@@ -14,32 +15,38 @@ class Lobby extends Component {
...
+              onChange={(ev) => this.setState({
+                player: { name: ev.target.value },
+              })}
+              onKeyPress={(e) => {
+                if (e.key === 'Enter') {
+                  this.save(e);
+                }
+              }}
+            />
+            <label> Press Enter, Any key</label>
```
Comment (mavirex) [inline client/components/Lobby.jsx:43]: This is a little misleading, a user can't press any key to move forward, right? Just Enter

File: client/components/index.jsx [hunk 4]
```diff
@@ -1,40 +1,72 @@
 import React, { Component } from 'react';
```
Comment (mavirex) [inline client/components/index.jsx:1]: I guess we can probably delete this whole file once this lobby functionality is up and running

File: client/components/Waitingroom.jsx [hunk 5]
```diff
@@ -15,8 +17,9 @@ class Waitingroom extends Component {
     const { socket } = this.props;
     socket.emit('change name', { name: socket.name });
     socket.on('goToGame', () => {
-      this.props.history.push('/imagegame');
+      this.state.game = true;
```
Comment (Djmiller1717) [inline client/components/Waitingroom.jsx]: this should be this.setState({ game: true })

File: client/components/Lobby.jsx (hunk 1 above)
Comment (00Halfman00) [reply client/components/Lobby.jsx]: Agreed.

File: client/components/Lobby.jsx (hunk 2 above)
Comment (00Halfman00) [reply client/components/Lobby.jsx]: Done. lol

File: client/components/Waitingroom.jsx (hunk 5 above)
Comment (00Halfman00) [reply client/components/Waitingroom.jsx]: That code is no more.

**Pull request closed without merging**